Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -143,6 +143,8 @@ public override void OnPreSceneQuery()
         /// <inheritdoc />
         public override void OnPostSceneQuery()
         {
+            base.OnPostSceneQuery();
```

Comment:
Interesting, sounds like we weren't doing this before. Was this a bug?

Reply:
It was fine because the base implementation was empty.